src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Utilities/Text/BitTextDemo.razor.scss

```
@@ -0,0 +1,54 @@
+﻿// The examples put loose runs of text of every variant next to each other, so they need a container that keeps a
+// consistent gap between them and stops the inline ones (a caption, an overline, a span) from sharing a line.
+.example-content {
+    gap: 0.5rem;
+    display: flex;
+    flex-flow: column;
+    align-items: flex-start;
+
+    // Every BitText is a component, so the scope attribute of this stylesheet never lands on it; the container that
+    // carries the attribute is what reaches them. An inline variant is given a line of its own here rather than a
+    // display of its own, so that the Block parameter still has something to demonstrate.
+    > ::deep .bit-txt {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)